ludobits/m/flow.lua

```diff
@@ -504,13 +512,30 @@ function M.until_flows(flows)
 	end
 
 	-- several coroutines
-	for k, v in pairs(flows) do
+	for _, instance in pairs(flows) do
 		M.until_true(function()
-			return coroutine.status(v.co) == "dead"
+			return coroutine.status(instance.co) == "dead"
 		end)
 	end
 end
 
+--- Wait until other flow coroutines with a specific group_id were finished
+-- @param group_id identifier of the flows group
+function M.until_group(group_id)
+	assert(group_id)
+	local group = {}
+
+	for _, instance in pairs(instances) do
+		if instance.group_id == ensure_hash(group_id) then
```

Comment:
Move ensure_hash(group_id) to outside the for loop.

```
group_id = ensure_hash(group_id)
for _, instance in pairs(instances) do
    if instance.group_id == group_id then
```

Reply:
Done!